content/en/docs/started/installing-kubeflow.md

```diff
@@ -122,6 +122,14 @@ Packaged distributions are developed and supported by their respective maintaine
         <td><a href="/docs/distributions/charmed/">Docs</a></td>
         <td><a href="https://charmed-kubeflow.io/docs/quickstart">External Website</a></td>
       </tr>
+      <tr>
+        <td>Kubeflow for Oracle Container Engine for Kubernetes</td>
+        <td>Oracle</td>
+        <td>Oracle Cloud Infrastructure (OCI)</td>
+        <td>1.6</td>
+        <td>N/A</td>
+        <td><a href="https://github.com/oracle-devrel/kubeflow-oke">External Website</a></td>
```

Comment:
Brand guideline: We shouldn't call something as `Kubeflow for XXX` 
![oracle](https://user-images.githubusercontent.com/37026441/189192156-ae246569-0e20-4f7c-9cbb-04d61027b39b.png)

Suggest to rename it to `Oracle Container Engine for Kubeflow` or `Kubeflow on Oracle Container Engine distribution`.

Reply:
Ok I update the title